actionmailbox/test/dummy/config/locales/en.yml

```diff
@@ -25,9 +25,11 @@
 #
 # en:
 #   'true': 'foo'
+#   'true': 'Yes'
+#   'false': 'No'
```

Comment:
I'm not sure we should duplicate the `true` key. Do we need lines 27 and 28? Can we just have one?

Reply:
I added that one to show the use-case I was running into, but happy to remove if it feels overly verbose.

Alternatively how about removing line 29 and changing line 28 to be the inverse of 27 (`'foo': 'true'`) to better illustrate that the key is irrelevant when the value is a truthy or falsey value?

Reply:
Can you please update the for you alternative suggestion? Thank you

Reply:
Sure thing. I've made that change. It didn't feel right to wrap `foo` (as a key) in single quotes, so I omitted them. They probably don't make since with the `foo` value above, but that seems like a different problem and is probably more opinion than fact.